build.sbt

```diff
@@ -339,8 +339,16 @@ lazy val executionJS = project.in(file("monix-execution/js"))
   .settings(requiredMacroDeps)
   .settings(executionCommon)
 
+lazy val doctestTestSettings = Seq(
+  testFrameworks += new TestFramework("utest.runner.Framework"),
```

Comment:
We can’t depend on utest, we’ll need Minitest integration.

I don’t think that’s too hard.

Reply:
I will see what it cost to integrate minitest with sbt-doctest

Reply:
first attempt to integrate minitest with sbt-doctest https://github.com/tkawachi/sbt-doctest/pull/108